patches/minecraft/net/minecraft/client/renderer/EntityRenderer.java.patch

```diff
@@ -66,7 +76,15 @@
                  GL11.glRotatef(entitylivingbase.field_70126_B + (entitylivingbase.field_70177_z - entitylivingbase.field_70126_B) * p_78467_1_ + 180.0F, 0.0F, -1.0F, 0.0F);
                  GL11.glRotatef(entitylivingbase.field_70127_C + (entitylivingbase.field_70125_A - entitylivingbase.field_70127_C) * p_78467_1_, -1.0F, 0.0F, 0.0F);
              }
-@@ -1052,7 +1057,9 @@
+@@ -583,6 +589,7 @@
+         }
+ 
+         GL11.glTranslatef(0.0F, f1, 0.0F);
++        MinecraftForge.EVENT_BUS.post(new OrientCameraEvent(field_78531_r, p_78467_1_));
```

Comment:
You should use fully qualified names to access classes, that way the patch becomes smaller because you don't need to import classes. Example: net.minecraftforge.client.event.OrientCameraEvent instead of OrientCameraEvent.